pex/pex.py

```diff
@@ -68,11 +68,16 @@ def _activate(self):
 
       # set up the local .pex environment
       pex_info = self._pex_info.copy()
+      combined_pex_paths = [pex_info.pex_path, self._vars.PEX_PATH]
       pex_info.update(self._pex_info_overrides)
+      pex_info.pex_path = ':'.join(filter(None, combined_pex_paths))
```

Comment:
I figure it makes more sense to use a temp var here rather than construct a new pex info API to preserve the overrides where we want pex_info vars overwritten. Doing something like `pex_info.merge_pex_paths_from_env(self._vars.PEX_PATH)` wouldn't be possible within this context as it would require more temp vars/code to ensure that oex_info.pex_path comes first in the unified path.

Reply:
`pex_info.merge_pex_paths_from_env(self._vars.PEX_PATH)` should totally work, and it would be a cleaner API here in particular because it doesn't involve you as a caller having to remember to put the content of `pex_info.pex_path` at the head before setting it - it would just merge it for you.

the implementation would look approximately like this:

```
from pex.util import merge_split

class PexInfo(...):
  ...
  def merge_pex_path(self, pex_path):
    """Merges a new PEX_PATH definition into the existing one (if any).
    :param string pex_path: The PEX_PATH to merge.
    """
    if not pex_path:
      return

    self.pex_path = ':'.join(merge_split(self.pex_path, pex_path))
```

bonus points if you provide a flag on `merge_split` to enable inline deduping.